### PR TITLE
HLF 2.x basic support

### PR DESCRIPTION
**Pull request by Hejwo**

### CODE REVIEW

File: e2e/__snapshots__/fabricaCommands.test.js.snap [hunk 1]
```diff
@@ -4,7 +4,7 @@ exports[`init should init simple fabrica config 1`] = `
 "{
   \\"$schema\\": \\"https://github.com/softwaremill/fabrica/releases/download/0.1.0-unstable/schema.json\\",
   \\"networkSettings\\": {
-    \\"fabricVersion\\": \\"1.4.6\\",
+    \\"fabricVersion\\": \\"2.2.1\\",
```
Comment (dzikowski) [inline e2e/__snapshots__/fabricaCommands.test.js.snap:7]: 💪

File: e2e-network/test-02-raft-2orgs.sh [hunk 2]
```diff
@@ -98,15 +98,17 @@ expectInvoke "cli.org2.com" "peer1.org2.com:7071" "my-channel2" "chaincode2" \
 
 # restart the network and wait for chaincodes
 (cd "$TEST_TMP" && "$FABRICA_HOME/fabrica.sh" stop && "$FABRICA_HOME/fabrica.sh" start)
-waitForChaincode "cli.org1.com" "peer1.org1.com:7061" "my-channel2" "chaincode1" "0.0.1"
-waitForChaincode "cli.org2.com" "peer1.org2.com:7071" "my-channel2" "chaincode1" "0.0.1"
+sleep 5
+waitForChaincode "cli.org1.com" "peer0.org1.com:7060" "my-channel1" "chaincode1" "0.0.1"
+waitForChaincode "cli.org2.com" "peer0.org2.com:7070" "my-channel1" "chaincode1" "0.0.1"
+sleep 5
```
Comment (dzikowski) [inline e2e-network/test-02-raft-2orgs.sh:104]: is `sleep` neccessary?

File: docs/schema.json [hunk 3]
```diff
@@ -42,7 +42,12 @@
             "1.4.8",
             "1.4.9",
             "1.4.10",
-            "1.4.11"
+            "1.4.11",
+            "2.0.1",
+            "2.1.0",
+            "2.1.1",
+            "2.2.0",
+            "2.2.1"
```
Comment (dzikowski) [inline docs/schema.json:50]: the newest version is `2.3.2`

File: generators/extend-config/configTransformers.js [hunk 4]
```diff
@@ -264,6 +264,7 @@ function transformNetworkSettings(networkSettingsJson) {
   return {
     ...networkSettingsJson,
     fabricCaVersion: getCaVersion(networkSettingsJson.fabricVersion),
+    fabricTwoDigitVersion: `${networkSettingsJson.fabricVersion.split('.').slice(0, 2).join('.')}`,
```
Comment (dzikowski) [inline generators/extend-config/configTransformers.js:267]: not used in in templates

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -12,7 +12,7 @@ jobs:
           SLACK_ICON: 'https://raw.githubusercontent.com/softwaremill/fabrica/issue-134/github-actions/logo-sygnet-192.png'
           SLACK_COLOR: '#ffd963'
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          MSG_MINIMAL: true
+          MSG_MINIMAL: ref
```
Comment (dzikowski) [inline .github/workflows/test.yml:15]: 👍

File: e2e/__snapshots__/network-01-simple.test.js.snap [hunk 6]
```diff
@@ -269,8 +243,9 @@ CHAINCODES_BASE_DIR=<absolute path>
 COMPOSE_PROJECT_NAME=<name with timestamp>
 LOGGING_LEVEL=info
 
-FABRIC_VERSION=1.4.6
-FABRIC_CA_VERSION=1.4.6
+FABRIC_VERSION=2.2.1
+FABRIC_CA_VERSION=1.5.0
+FABRIC_TWO_DIGIT_VERSION=2.2.1
```
Comment (dzikowski) [inline e2e/__snapshots__/network-01-simple.test.js.snap:248]: ?

File: samples/fabricaConfig-1org-1channel-1chaincode.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://github.com/softwaremill/fabrica/releases/download/0.1.0-unstable/schema.json",
   "networkSettings": {
-    "fabricVersion": "1.4.6",
+    "fabricVersion": "2.2.1",
```
Comment (dzikowski) [inline samples/fabricaConfig-1org-1channel-1chaincode.json:4]: need to update `TEST_CASES.md` as well

File: samples/fabricaConfig-2orgs-2channels-2chaincodes-tls-raft.json [hunk 8]
```diff
@@ -99,7 +99,7 @@
       "name": "chaincode1",
       "version": "0.0.1",
       "lang": "node",
-      "channel": "channel2",
+      "channel": "channel1",
```
Comment (dzikowski) [inline samples/fabricaConfig-2orgs-2channels-2chaincodes-tls-raft.json:102]: 👍

**Review by dzikowski — APPROVED** [review]

File: e2e-network/test-02-raft-2orgs.sh (hunk 2 above)
Comment (Hejwo) [reply e2e-network/test-02-raft-2orgs.sh:104]: no - was an old garbage

File: e2e/__snapshots__/network-01-simple.test.js.snap (hunk 6 above)
Comment (Hejwo) [reply e2e/__snapshots__/network-01-simple.test.js.snap:248]: yeah, not needed

**Pull request merged**